clients/js/cmds/aptos.ts

```diff
@@ -55,24 +55,17 @@ function assertNetwork(n: string): asserts n is Network {
 }
 
 exports.command = 'aptos';
-exports.desc = 'Aptos utilities ';
+exports.desc = 'Aptos utilities';
 exports.builder = function(y: typeof yargs) {
   return y
     .command("init-token-bridge", "Init token bridge contract", (yargs) => {
       return yargs
         .option("network", network_options)
         .option("rpc", rpc_description)
-        // TODO(csongor): once the sdk has this, just use it from there
-        .option("contract-address", {
-          alias: "a",
-          required: true,
-          describe: "Address where the wormhole module is deployed",
-          type: "string",
-        })
     }, async (argv) => {
       const network = argv.network.toUpperCase();
       assertNetwork(network);
-      const contract_address = evm_address(argv["contract-address"]);
+      const contract_address = evm_address(CONTRACTS[network].aptos.core);
```

Comment:
these slipped past code review it seems, but this now only works for the core contract -- instead, these commands should take the module as an argument. I'll revert this for now for the nft bridge

Reply:
Ahh good catch, will fix